### PR TITLE
fix CSS cm-editor * + dark:secondary-text + add new compact UI

### PR DESCRIPTION
**Pull request by versecafe**
image

> Some extra CSS changes to add warning and error tooltip backgrounds in followup PR

### CODE REVIEW

File: packages/web/src/index.css [hunk 1]
```diff
@@ -37,6 +38,7 @@
     --sb-red-20: 0 100% 86%;
     --sb-red-30: 0 100% 76%;
     --sb-red-40: 0 100% 66%;
+    --sb-red-40-subtle: 0 100% 66% 0.1;
```
Comment (benjreinhart) [inline packages/web/src/index.css:41]: Why are the existing colors not sufficient here?

Comment (versecafe) [reply packages/web/src/index.css]: needed alpha channel variant, got rid of these and moved to `hsl(var(--sb-*-*) / 0.1)`

File: packages/web/src/index.css [hunk 2]
```diff
@@ -390,23 +394,28 @@
...
 }
 
 .cm-tooltip {
   @apply overflow-hidden;
   @apply !bg-background;
   @apply border;
-  @apply rounded-sm;
   @apply shadow-md;
   @apply !border-border;
+  font-size: 14px;
```
Comment (nichochar) [inline packages/web/src/index.css:409]: isn't 14px the same size as the editor? In which case this is not useful. Didn't we want it to be smaller?

Comment (versecafe) [reply packages/web/src/index.css]: it gets override by tw anyways but just pulled it since it's empty junk

**Comment by nichochar** [comment]
Code LGTM, but idk about the screenshot line-height. I'll take over the PR and get this through, since it's getting subjective.

**Comment by nichochar** [comment]
OK, came around after playing around with tighter line-heights, this is actually good as is. Merging!

**Review by nichochar — APPROVED** [review]

**Pull request merged**